heidrun/dlg_dc.rb

```diff
@@ -0,0 +1,74 @@
+def dlg_preview(identifier)
+  id = identifier.rpartition(':').last.split('_', 3)
+  return nil if id.count != 3
+  "http://dlg.galileo.usg.edu/#{id[0]}/#{id[1]}/do-th:#{id[2]}"
+end 
+
+Krikri::Mapper.define(:dlg_dc, :parser => Krikri::OaiDcParser) do
+  provider :class => DPLA::MAP::Agent do
+    uri 'http://dp.la/api/contributor/georgia'
+    label 'Digital Library of Georgia'
+  end
+
+  dataProvider :class => DPLA::MAP::Agent do
+    providedLabel record.field('dc:contributor')
+  end
+
+  isShownAt :class => DPLA::MAP::WebResource do
+    uri record.field('dc:identifier')
+  end
+
+  preview :class => DPLA::MAP::WebResource do
+    uri dlg_preview(record.field('xmlns:header', 'xmlns:identifier').first)
```

Comment:
After dpla/krikri#115, this becomes `header.field('xmlns:identifier').first_value.map { |i| dlg_preview(i) }`